src/lib/Organism/Overview.svelte

```diff
@@ -312,17 +300,16 @@ color: var(--yellow);
 .list-btn:focus ~ .grid-container .grid-item {
     display: flex;
     flex-direction: column;
-    /* padding: 0 20px; */
 }
 
 .list-btn:focus ~ .grid-container .grid-item img {
     width: 100%;
-    height: auto;
+    height: 100%;
 }
  
 
 .list-btn:focus ~ .grid-container .image-container img {
-    height: auto;
+    height: 100%;
 }
```

Comment:
De images in de carousel hebben geen consitente aspect ratio. Wellicht kan je dit voor elkaar krijgen met de aspect-ratio en/of object-fit CSS properties

![image](https://github.com/fdnd-agency/crossmarx/assets/1061632/245aea9f-e8dd-494f-98bb-b4ddb21b9632)
![image](https://github.com/fdnd-agency/crossmarx/assets/1061632/57922d31-22a4-4925-8fbe-254e83b70c9f)

Wellicht kan je ook proberen om bij een smaller scherm de foto en het tekstgedeelte verticaal te plaatsen. Dan heb je meer ruimte voor de content en zal het formaat foto in verschillende contexten minder verschillen in de breedte